package.json

```diff
@@ -18,13 +18,17 @@
     "lint-staged": "lint-staged",
     "clean": "del-cli helpers bin",
     "lint": "run-p -c --aggregate-output lint-ts",
-    "pretest": "run-p -c --aggregate-output lint validate-filenames lint-staged",
-    "test": "run-p --aggregate-output -l check-ts"
+    "pretest": "run-p -c --aggregate-output lint validate-filenames lint-staged 'clown check'",
+    "test": "run-p --aggregate-output -l check-ts",
+    "clown": "clown",
+    "pre-push": "yarn test",
+    "pre-commit": "yarn pretest",
+    "tslint": "yarn lint-ts"
```

Comment:
Some of the tasks here are overridden at `clownOverrides/package.json`. To see where these configurations come from, you can look at `clown.json` and the folders inside `@hollowverse/config/clown/`.

The end result is not the cleanest, but the more we standardize the tasks between our repos, the cleaner the end result will be.

Reply:
> The end result is not the cleanest

Still much cleaner (and composable) than what we had.

> the more we standardize the tasks between our repos, the cleaner the end result will be

Agreed.